resources/sdk/purecloudjavascript/templates/README.mustache

```diff
@@ -76,7 +76,23 @@ The Client Credentials grant only works when used in node.js. This is restricted
 
 ```{"language":"javascript"}
 const client = {{moduleName}}.ApiClient.instance;
-client.loginClientCredentialsGrant(clientId, clientSecret)
+client.loginSaml2Bearer(clientId,clientSecret,orgName,encodedSamlAssertion)
```

Comment:
1. This indentation is wrong and inconsistent
2. This is supposed to be the client credentials example